app/refactor/views/p4/tad/components/table_2/top_right_block.scala.xml

```diff
@@ -97,7 +97,7 @@
                         <fo:block margin-top="1mm" font-weight="bold">
                             <fo:inline >
                                 <fo:block>
-                                    @consignmentPreviousDocuments
+                                    @consignmentPreviousDocuments.toList.mkString("\u200b")
```

Comment:
Again, could this be `@block(consignmentPreviousDocuments)` ? Or does a block-container here mess up the formatting?